ui/features/landing/landing.controller.js

```diff
@@ -14,10 +14,11 @@ export default class LandingController {
     this.$rootScope = $rootScope;
     this.model = null;
     this.neighbors = [];
-    var i = 0;
+    var color = {'subClassOf':'#FF4646','~subClassOf':'#992A2A','isDefinedBy':'#00FF00','~isDefinedBy':'#005B00','equivalentClass':'#5BCBB6','~equivalentClass':'#2D655B','sameAs':'#774177','subPropertyOf':'#FFC04C','!subPropertyOf':'#A47B30','equivalentProperty':'#6666FF','~equivalentProperty':'#3D3D99','property':'#FF7032','~property':'#CC3D00','type':'#B2FFFF','~type':'#EDD95D','operand':'#000000'};
     var duration = 750;
```

Comment:
For code maintainability purposes, please format objects and arrays with one element per line. Massively improves the ability of git to deal with multiple maintainers and diffs. So the above should be:
var color = {
  'subClassOf': '#FF4646',
  '~subclassOf': ....
};